app/both/collections/activityReportAggregate.js

```diff
@@ -0,0 +1,61 @@
+import SimpleSchema from 'simpl-schema';
+
+ActivityReportAggregate = new Mongo.Collection(
+  'activityReportAggregate'
+);
+
+/* The activity count and minute data */
+const countsSchema = new SimpleSchema({
+  activity_count: {
+    type: Number,
+  },
+  activity_minutes: { type: Number },
+});
+
+/* Schema of object inside the array */
+const valuesSchema = new SimpleSchema({
+  key: {
+    type: Date,
+  },
+  value: {
+    type: countsSchema,
+  },
+});
+
+/* Schema of the monthly/weekly data array */
+const aggregateSchema = new SimpleSchema({
+  key: {
+    type: String,
+  },
+  values: {
+    type: Array,
+  },
+  'values.$': {
+    type: valuesSchema,
+  },
+});
+
+/* Collection schema as per the object returned by mongo query.
+   Each record contains aggregated data for both time granularities i.e. monthly and weekly
+*/
+ActivityReportAggregate.Schema = new SimpleSchema({
+  Date: {
```

Comment:
How is this `Date` field used? It seems like the date values would be in the `weeklyData` and `monthlyData` array items.

If this is for the `updated_date,` let's be more explicit about the purpose of the field by giving it a more semantic name. Using a more semantic name would also avoid colliding with the `Date` class.